app/routes.py

```
@@ -3,12 +3,13 @@
 import asyncio
 import uuid
 from typing import Any, Dict, List, Optional, Tuple
+from datetime import datetime
 
 from fastapi import APIRouter, HTTPException, Request, status
 from fastapi.responses import JSONResponse
 from pydantic import BaseModel, Field, HttpUrl
```

Ruff (0.8.2)

`pydantic.BaseModel` imported but unused

Remove unused import

(F401)